content/en/docs/concepts/cluster-administration/dra.md

```diff
@@ -0,0 +1,180 @@
+* Running pods will continue to run.
+
+### Confirm your DRA driver exposes a liveness probe and utilize it
+
+Your DRA driver likely implements a grpc socket for healthchecks as part of DRA
+driver good practices. The easiest way to utilize this grpc socket is to
+configure it as a liveness probe for the DaemonSet deploying your DRA driver.
+Your driver's documentation or deployment tooling may already include this, but
+if you are building your configuration separately or not running your DRA driver
+as a Kubernetes pod, be sure that your orchestration tooling restarts the DRA
+driver on failed healthchecks to this grpc socket. Doing so will minimize any
+accidental downtime of the DRA driver and give it more opportunities to self
+heal, reducing scheduling delays or troubleshooting time.
+
+### When draining a node, drain the DRA driver as late as possible
+
+The DRA driver is responsible for unpreparing any devices that were allocated to
+Pods, and if the DRA driver is drained before Pods with claims have been
+deleted, it will not be able to finalize its cleanup. If you implement custom
+drain logic for nodes, consider checking that there are no allocated/reserved
+ResourceClaim or ResourceClaimTemplates before terminating the DRA driver
+itself.
+
+
+## Monitor and tune components for higher load, especially in high scale environments
+
+Control plane component `kube-scheduler` and the internal ResourceClaim
+controller orchestrated by the component `kube-controller-manager` do the heavy
+lifting during scheduling and binding of Pods with claims based on metadata
+stored in the DRA APIs. Compared to non-DRA scheduled Pods, the number of API
+server calls, memory, and CPU utilization needed by these components is
+increased for Pods using DRA claims. In addition, node local components like the
+DRA driver and kubelet utilize DRA APIs to determine what hardware to allocate
+at Pod sandbox creation time. Especially in high scale environments where
+clusters have many nodes, and/or deploy many workloads that heavily utilize
+DRA defined resource claims, the cluster administrator should configure the
+relevant components to anticipate the increased load. 
+
+The effects of mistuned components can have direct or snowballing affects
+causing different symptoms during the Pod lifecycle. If the `kube-scheduler`
+component's QPS and burst configurations are too low, the scheduler might
+quickly identify a suitable node for a Pod but take longer to bind the Pod to
+that node. During Pod startup, the QPS and Burst parameters in the client-go
+configuration within `kube-controller-manager` are critical.
+
+The specific values to tune your cluster to depend on a variety of factors. For
+example, a 100-node test involving 720 long-lived pods (90% saturation) and 80
+churn pods (10% churn, 10 times), with a job creation QPS of 10, necessitated
+setting kube-controller-manager QPS to 200 and Burst to 400 to meet equivalent
```

Comment:
lower bound qps: 75 and burst: 150.

with these values, we see that the client side rate limiter is triggered so it is enough to protect apiserver from explosive burst but it is high enough that pod startup SLOs were not impacted.

Reply:
THANK YOU!!!!